### ISSUE TITLE
How can we set "android.software.leanback" to "false" in Android TV's manifest file?

### ISSUE THREAD

**Issue by SandeepKharbanda** [pavjacko/renative#570]
Hi,

Currently, in my Android TV project, our **AndroidManifest.xml** has **"android.software.leanback"** to **"true"** i.e
```
<uses-feature
      android:name="android.software.leanback"
      android:required="true"
    />
```
And, I need to change its value to **"false"** and add the below code in the **renative.json** file i.e

```
"androidtv": {
      "version": "1.2",
      "versionCode": 3,
      "compileSdkVersion": 29,
      "targetSdkVersion": 29,
      "gradle.properties": {
        "android.useDeprecatedNdk": true
      },
      "AndroidManifest": {
        "children": [
          {
            "tag": "application",
            "android:name": ".MainApplication",
            "android:usesCleartextTraffic": true,
            "android:largeHeap": true,
            "tools:targetApi": 28
          },
          {
            "tag": "uses-feature",
            "android:name": "android.software.leanback",
            "android:required": false            
          }
        ]
      }
    }
```
But, it doesn't change its value to **"false"** in **AndroidManifest.xml**. Could you please help me out with this what am I doing wrong here? 

Any help would be appreciated.

**Comment by SandeepKharbanda** [comment]
Any update?

**Comment by pavjacko** [comment]
@SandeepKharbanda sorry, swamped last few weeks. I can confirm top-level tags outside application not being merged into final AndroidManifest. fix coming up soon

**Comment by pavjacko** [comment]
@SandeepKharbanda should fixed in latest alpha `0.31.0-alpha.25` release. official `0.31.0` will be out this week
